src/libs/localFileDownload/index.ios.ts

```diff
@@ -1,16 +1,14 @@
 import {Share} from 'react-native';
 import RNFetchBlob from 'react-native-blob-util';
 import * as FileUtils from '../fileDownload/FileUtils';
+import LocalFileDownload from './types';
 
 /**
  * Writes a local file to the app's internal directory with the given fileName
  * and textContent, so we're able to share it using iOS' share API.
  * After the file is shared, it is removed from the internal dir.
- *
- * @param {String} fileName
- * @param {String} textContent
  */
-export default function localFileDownload(fileName, textContent) {
+export default function localFileDownload({fileName, textContent}: LocalFileDownload) {
```

Comment:
It's not relevant anymore given the suggestion I gave on `types.ts` file, but let's be careful with this. You changed the signature of the file to accept a object instead of two parameters, that would break the app when some file use this functions.

Reply:
Ayye! all fixed now, Thanks @fabioh8010.